lib/game.dart

```diff
@@ -31,11 +32,17 @@ class CoronaJump extends BaseGame with HasWidgetsOverlay {
     // background
     add(background);
 
-    // start menu
-    showMenuOverlay();
+    // landingpage
+    showLandingPage();
+  }
+
+  void showLandingPage() {
+    addWidgetOverlay(
```

Comment:
LandingPage should not be an overlay on top of Flame, because effectively the LandingPage is not an overlay. It is separate from the game.
Instead, LandingPage (and CoronaJump (game.dart)) should be Flutter routes with navigation enabled between them through Flutter navigation logic: https://flutter.dev/docs/cookbook/navigation/navigation-basics

Reply:
I agree. This will take me a lot of time to figure out tho. 
Why not keeping it that way until we decide if corona jump has a future or not? Otherwise its unused work

Reply:
ok